index.js

```diff
@@ -50,4 +50,8 @@ OpenDKIM.prototype.tmpdir = function (data) {
   });
 };
 
+OpenDKIM.prototype.lib_feature = function (feature) {
+  this.lib_feature_obj({feature: feature});
```

Comment:
I might end up doing this `*_obj` for other native methods too.  I'm looking into making a bunch of the interface async with Promises.